content/en/agent/kubernetes/_index.md

```diff
@@ -255,6 +254,100 @@ For further details on setting up Operator, including information about using to
       fsGroup: <DOCKER_GROUP_ID>
 ```
 
+## Kubernetes resources for Live Containers
+
+The Agent and [Cluster Agent][3] can be configured to retrieve Kubernetes resources for [Live Containers][4]. This feature allows you to monitor the state of pods or deployments in a specific namespace or availability zone, view resource specifications for failed pods within a deployment, correlate node activity with related logs, and more.
+
+Kubernetes resources for Live Containers requires the following set up prior to configuration:
+
+- Agent version >= 7.21.1
+- Process-agent is running (it doesn't need to run the process collection), and configured with the following options:
+
+  {{< code-block lang="yaml">}}
+  - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
+    value: "true"
+  - name: DD_ORCHESTRATOR_CLUSTER_ID
+    valueFrom:
+    configMapKeyRef:
+      name: datadog-cluster-id
+      key: id
+  {{< /code-block >}}
+
+  It must also be configured to communicate with the Cluster Agent, see the [Cluster Agent Setup documentation][5] for more information about this.
+
+- Cluster Agent version >= 1.9.0
+- Cluster Agent is running, and the Agent must be able to communicate with it
```

Comment:
All this block should be part of the DaemonSet tab, as the helm chart will take care of all of this automatically if it's used.